scripts/install/provision-spinnaker.md

```diff
@@ -9,12 +9,23 @@ Select the project in which you'll install Spinnaker, then click **Start**, belo
 
 ## Spinnaker Installation
 
-Now let's provision Spinnaker within {{project-id}}.
+Click the copy/paste button on the commands below to copy each command to your shell,
+then press **Enter** to run each commmand.
+
+### Configure Git
 
-Click the copy/paste button on the command below to copy that command to your shell,
-then press **Enter** to run it from the Shell.
+If you haven't already configured Git, please use the commands below to do so now.
+Replace `[EMAIL_ADDRESS]` with your Git email address, and replace `[USERNAME]`
+with your Git username.
 
-### Configure the environment.
+```bash
+git config --global user.email "[EMAIL_ADDRESS]"
+git config --global user.name "[USERNAME]"
+```
+
+### Configure the environment
+
+Now let's provision Spinnaker within {{project-id}}.
```

Comment:
s/within/within your project/
I know it seems lame but sometimes people won't know.

Reply:
Done.